### ISSUE TITLE
Not work with foundry 12

### ISSUE THREAD

**Issue by gandb** [xaukael/pointers#4]
Ocorrus this error:
```
pointers.js:36 Uncaught TypeError: Cannot read properties of undefined (reading 'pointer')
    at pointerSocket.emit (pointers.js:36:172)
    at Object.onDown (pointers.js:99:44)
    at KeyboardManager._executeKeybind (foundry.js:2439:66)
    at KeyboardManager._processKeyboardContext (foundry.js:2480:33)
    at KeyboardManager._handleKeyboardEvent (foundry.js:2538:10)
    at foundry.js:2152:54
```

**Comment by xaukael** [comment]
fixed in 0.0.6 and compatibility updated to v12

**Comment by gandb** [comment]
Thanxs

I tested and now worked perfect, thank you
